src/core/collision.hpp

```diff
@@ -162,4 +162,19 @@ inline double collision_detection_cutoff() {
   return 0.;
 }
 
+/** @brief Whether or not the verlet criterion mathes particle types */
+inline bool collision_detection_match_part_type() {
+  return collision_params.mode == COLLISION_MODE_GLUE_TO_SURF;
+}
+
+/** @brief first particle type for verlet criterion */
+inline int collision_detection_part_type1() {
```

Comment:
why those one line functions and why do the have a name that doesn't match the variable they are returning?